dali/base/dadfs.cpp

```diff
@@ -9567,7 +9571,7 @@ class CDaliDFSServer: public Thread, public CTransactionLogTracker, implements I
         StringArray authScopes;
         CIArrayOf<CFileMatch> matchingFiles;
         start = msTick();
-        count = scanner.getResults(auth, udesc, matchingFiles, authScopes);
+        count = scanner.getResults(auth, udesc, matchingFiles, authScopes, false);
```

Comment:
That OOM problem is due to: cSort() : buf(0x100000*100,0x10000,true). But, a lot of memory is used before sorting.

ESP does make calls that end up calling iterateFiles. Those calls do not need a sorted file list. I can check the file limit inside the iterateFiles(). But, I am not sure whether that will break anything or not since the iterateFiles is also called from non-esp servers (such as dali, sasha, etc.).